onboarding_documentation/technical_documentation/metamist_docker/metamist_docker_tutorial.md

```diff
@@ -0,0 +1,763 @@
+# Introduction
+
+The aim of the following exercise is to allow you to get first hand experience working with the CPG codebase, the tools used to conduct analyses, as well as the supporting infrastructure that enables the management and utilisation of data.
+
+## Table of Contents
+
+1. [Query metamist for `.fastq` files](#1-query-metamist-for-fastq-files)
+2. [Build and publish a Docker image](#2-build-and-publish-a-docker-image)
+3. [Write a script to run FastQE](#3-write-a-script-to-run-fastqe)
+4. [Update Metamist with the location of the output files](#4-update-metamist-with-the-location-of-the-output-files)
+5. [Run the jobs through analysis runner](#5-run-the-jobs-through-analysis-runner)
+6. [Pull the output and visualise](#6-pull-the-output-and-visualise)
+
+## 1. Query metamist for `.fastq` files
+
+The first step in understanding Metamist is to understand how to query the database so that we can get the necessary information on our samples. Metamist is used to track file locations, participant IDs, statuses of samples (i.e. the location of the files on Google Cloud, whether the the `.fastq` files have been aligned to form`.cram`s, whether there is a `.vcf` file available, and whether the sample has been joint called with other samples). Metamist is CPG's way of keeping track of everything.
```

Comment:
Suggest defining metamist clearly as CPG's metadata db and providing a link to the documentation. 

What is 'everything' when we are "Keeping track of everything"?